### ISSUE TITLE
Remove forced dependency to old Vert.x web and api-contract library

### ISSUE THREAD

**Issue by ppatierno** [strimzi/strimzi-kafka-bridge#383]
When updating the Vert.x version to 3.8.5 (mostly for the Vert.x Kafka client) we are facing the following error with some producer tests where the value is sent as a string:

```
"Validation error on: body.records[0].value - $.records[0].value: string found, object expected"
```

but at the same time the OpenAPI v3 definition for the producer record contains:

```json
                  "value": {
                        "oneOf": [
                            {
                                "type": "object"
                            },
                            {
                                "type": "string"
                            }
                        ]
                    },
```

which shows that value can be a string as well.
This contract is working fine on Vert.x 3.8.1 but doesn't work anymore with upper versions.
Maybe it's a validation bug in the OpenAPI contract library?
For now we are forcing vertx-web and vertx-api-contract to 3.8.1 but we should bump to 3.8.5 as the rest of Vert.x components.

This issue is related to #380 and #382.

/cc @slinkydeveloper

**Comment by slinkydeveloper** [comment]
Can you try to create a minimum reproducer and open a ticket on vertx-web repo?

**Comment by ppatierno** [comment]
Fixed by #397

**Comment by ppatierno** [comment]
Close as fixed per previous comment.
